src/ZTUtils/Zope.py

```diff
@@ -188,6 +203,9 @@ def make_query(*args, **kwargs):
     qlist = complex_marshal(list(d.items()))
     for i in range(len(qlist)):
         k, m, v = qlist[i]
+        if type(v) == unicode:
+            _default_encoding()
+            v = v.encode(_DEFAULT_ENCODING)
```

Comment:
I think it would be nicer if `_default_encoding()` returns `_DEFAULT_ENCODING`. This makes the code more readable IMHO as the function has to be called anyway.